recensio/theme/skins/recensio_theme/searchNoVolDoubles.pt

```diff
@@ -228,10 +228,10 @@
                       >
 
                     <span class="documentModified">
-                        <span i18n:translate="box_last_modified">
-                        last modified
+                        <span i18n:translate="box_effective">
+                        published
                         </span>
-                        <span tal:replace="python:toLocalizedTime(result.ModificationDate, long_format=1)">
+                        <span tal:replace="python:toLocalizedTime(result.ModificationDate)">
```

Comment:
The text now claims that this is the publication date, but it's actually still the modification date.